processor/stream/processor.go

```diff
@@ -46,6 +46,14 @@ var (
 
 const (
 	batchSize = 10
+
+	errorEventType            = "error"
```

Comment:
aren't this constants defined already elsewhere? can we reuse them? if there is a circular dependency maybe move those constants to neutral territory?

Reply:
AFAIK they're not defined elsewhere. There are other constants with the value "transaction", "error", etc. -- but they have different meanings. This is the only place (AFAIK) where we define the top-level keys for the ND-JSON event stream.